From ecb7e47a1484242a9ebb5c586461dc4ba28c2353 Mon Sep 17 00:00:00 2001
From: Dentrax <furkan.turkal@chainguard.dev>
Date: Sun, 31 Aug 2025 17:45:47 +0300
Subject: [PATCH] x-pack: add bcmail-fips entitlement

watcher plugin requires access to bcmail-fips jar, thus grant it.

Signed-off-by: Dentrax <furkan.turkal@chainguard.dev>
Co-authored-by: Dimitri John Ledkov <dimitri.ledkov@surgut.co.uk>
---
 .../watcher/src/main/plugin-metadata/entitlement-policy.yaml   | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/x-pack/plugin/watcher/src/main/plugin-metadata/entitlement-policy.yaml b/x-pack/plugin/watcher/src/main/plugin-metadata/entitlement-policy.yaml
index 0b26b58e..fe4bbcdd 100644
--- a/x-pack/plugin/watcher/src/main/plugin-metadata/entitlement-policy.yaml
+++ b/x-pack/plugin/watcher/src/main/plugin-metadata/entitlement-policy.yaml
@@ -13,4 +13,5 @@ ALL-UNNAMED:
     - relative_path: ".mailcap"
       relative_to: "home"
       mode: "read"
-
+    - path: "/usr/share/elasticsearch/lib/bcmail-fips.jar"
+      mode: "read"
-- 
2.39.5 (Apple Git-154)

